priv/repo/migrations/20210207173118_add_token_balance_attendee.exs

```diff
@@ -0,0 +1,10 @@
+defmodule Safira.Repo.Migrations.AddTokenBalanceAttendee do
+  use Ecto.Migration
+
+  def change do
+
+    alter table(:attendees) do
+      add :token_balance, :integer
+  end
+end
+end
```

Comment:
You should run `priv/repo/migrations/20210207173118_add_token_balance_attendee.exs`